src/Registers.cc

```diff
@@ -91,7 +91,7 @@ template <> struct RegisterInfo<rr::X64Arch> {
   static const size_t num_registers = DREG_NUM_LINUX_X86_64;
   typedef RegisterTable<num_registers> Table;
   static Table registers;
-  static RegisterValue non_gdb_registers[2];
+  static RegisterValue non_gdb_registers[0];
```

Comment:
With this patch there are no longer any `non_gdb_registers` so you should just remove all related code.